features/app-launch-handler.yml

```diff
@@ -0,0 +1,13 @@
+name: Launch handlers
+description: The `launch_handler` and `client_mode` web app manifest members configure how web app launches should behave when an instance of the app is already open. For example, you can choose to focus on an existing app instead of launching a new one.
+spec: https://wicg.github.io/web-app-launch/
+group: progressive-web-app
+compat_features:
+  - api.LaunchParams
+  - api.LaunchParams.files
```

Comment:
This BCD key is already used by this feature: [app-file-handlers](https://web-platform-dx.github.io/web-features-explorer/features/app-file-handlers/), which is about the specific case of handling files. Let's remove it from this file.

```suggestion
```